internal/runners/export/apikey.go

```diff
@@ -42,6 +42,8 @@ type primeable interface {
 	primer.Outputer
 	primer.Configurer
 	primer.Projecter
+	primer.Analyticer
```

Comment:
Since `primeable` is now used elsewhere in this package, please hoist it out into a more generic file, maybe export.go? (Not sure exactly).